tt-xla uplift failed after runtime changes

After commit https://github.com/tenstorrent/tt-mlir/commit/dacc0ae77af0cdc61725bbd102e8aeae41ce8ecb tt-xla no longer builds, with the following error:

```
13/31] Building CXX object src/common/CMakeFiles/TTPJRTCommon.dir/pjrt_implementation/executable_instance.cc.o
FAILED: [code=1] src/common/CMakeFiles/TTPJRTCommon.dir/pjrt_implementation/executable_instance.cc.o 
/usr/bin/c++  -I/localdev/sgligorijevic/tt-xla-2/src -I/localdev/sgligorijevic/tt-xla-2/inc -I/localdev/sgligorijevic/tt-xla-2/third_party/loguru/src/loguru-install/include -I/localdev/sgligorijevic/tt-xla-2/third_party/pjrt_c_api -I/localdev/sgligorijevic/tt-xla-2/third_party/tt-mlir/install/include -I/opt/ttmlir-toolchain/include -I/opt/ttmlir-toolchain/src/shardy -I/opt/ttmlir-toolchain/src/stablehlo -I/localdev/sgligorijevic/tt-xla-2/build/_deps/protobuf-src/src -fno-rtti -O3 -DNDEBUG -std=gnu++17 -fPIC -MD -MT src/common/CMakeFiles/TTPJRTCommon.dir/pjrt_implementation/executable_instance.cc.o -MF src/common/CMakeFiles/TTPJRTCommon.dir/pjrt_implementation/executable_instance.cc.o.d -o src/common/CMakeFiles/TTPJRTCommon.dir/pjrt_implementation/executable_instance.cc.o -c /localdev/sgligorijevic/tt-xla-2/src/common/pjrt_implementation/executable_instance.cc
In file included from /localdev/sgligorijevic/tt-xla-2/third_party/tt-mlir/install/include/tt/runtime/types.h:15,
                 from /localdev/sgligorijevic/tt-xla-2/inc/common/pjrt_implementation/executable_image.h:22,
                 from /localdev/sgligorijevic/tt-xla-2/inc/common/pjrt_implementation/executable_instance.h:18,
                 from /localdev/sgligorijevic/tt-xla-2/src/common/pjrt_implementation/executable_instance.cc:11:
/localdev/sgligorijevic/tt-xla-2/third_party/tt-mlir/install/include/tt/runtime/utils.h:18:10: fatal error: tt/runtime/flatbuffer/flatbuffer.h: No such file or directory
   18 | #include "tt/runtime/flatbuffer/flatbuffer.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

--- a/runtime/lib/CMakeLists.txt
+++ b/runtime/lib/CMakeLists.txt
@@ -76,7 +76,9 @@ if (TTMLIR_ENABLE_RUNTIME)
       DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}
       COMPONENT SharedLib
   )
-  install(DIRECTORY ${PROJECT_BINARY_DIR}/runtime/include/tt/runtime/flatbuffer
+  install(DIRECTORY
+    ${PROJECT_SOURCE_DIR}/runtime/include/tt/runtime/flatbuffer
+    ${PROJECT_BINARY_DIR}/runtime/include/tt/runtime/flatbuffer
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/tt/runtime
     COMPONENT SharedLib
     FILES_MATCHING

